recipes/physx/4.x.x/conanfile.py

```diff
@@ -0,0 +1,327 @@
+import os
+import shutil
+import time
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class PhysXConan(ConanFile):
+    name = "physx"
+    description = "The NVIDIA PhysX SDK is a scalable multi-platform " \
+                  "physics solution supporting a wide range of devices, " \
+                  "from smartphones to high-end multicore CPUs and GPUs."
+    license = "BSD-3-Clause"
+    topics = ("conan", "PhysX", "physics")
+    homepage = "https://github.com/NVIDIAGameWorks/PhysX"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake"
+    settings = "os", "compiler", "arch", "build_type"
+    short_paths = True
+    no_copy_source = True
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "release_build_type": ["profile", "release"],
```

Comment:
I think this option should be a boolean: ``"profile_build_type": [True, False],``, as no one should be settings this option to "release"